Runtime/Scripts/GLTFSceneImporter.cs

```diff
@@ -1101,6 +1102,41 @@ async Task CreateNodeComponentsAndChilds(bool ignoreMesh = false, bool onlyMesh
 			progress?.Report(progressStatus);
 		}
 
+		private void ConstructCamera(GameObject nodeObj, Node node)
+		{
+			if (node.Camera == null)
+				return;
+
+			if (_options.CameraImport == CameraImportOption.None)
+				return;
+			
+			var camera = node.Camera.Value;
+			Camera unityCamera = null;
+			if (camera.Orthographic != null)
+			{
+				unityCamera = nodeObj.AddComponent<Camera>();
+				unityCamera.orthographic = true;
+				unityCamera.orthographicSize = Mathf.Max((float)camera.Orthographic.XMag, (float)camera.Orthographic.YMag);
+				unityCamera.farClipPlane = (float)camera.Orthographic.ZFar;
+				unityCamera.nearClipPlane = (float)camera.Orthographic.ZNear;
+			}
+			else
+			if (camera.Perspective != null)
+			{
+				unityCamera = nodeObj.AddComponent<Camera>();
+				unityCamera.orthographic = false;
+				unityCamera.fieldOfView = (float)camera.Perspective.YFov * Mathf.Rad2Deg;
+				unityCamera.farClipPlane = (float)camera.Perspective.ZFar;
+				unityCamera.nearClipPlane = (float)camera.Perspective.ZNear;
+			}
+
+			if (!unityCamera)
+				return;
+			
+			if (_options.CameraImport == CameraImportOption.ImportAndCameraDisabled)
+				unityCamera.enabled = false;
```

Comment:
This _may_ conflict with other extensions – e.g. imagine one that stores "enabled/disabled" state for GameObjects; could that run after camera generation so that the node can be enabled again / set to the state that the extension wants?